packages/esm-patient-chart-app/src/visit/visit-form/visit-type.component.test.tsx

```diff
@@ -13,6 +13,7 @@ const mockHandleChange = jest.fn();
 const mockGoToPage = jest.fn();
 
 jest.mock('@openmrs/esm-framework', () => ({
+  ...(jest.requireActual('@openmrs/esm-framework') as any),
```

Comment:
Could you add a default mock for `useVisitTypes` to [esm-framework](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-framework/mock.tsx) so we can get rid of all these?